core/src/test/java/org/apache/calcite/test/enumerable/EnumerableHashJoinTest.java

```diff
@@ -125,14 +124,12 @@ class EnumerableHashJoinTest {
         .query(
             "select e.empid, e.name, d.name as dept from emps e right outer "
                 + "join depts d on e.deptno=d.deptno and e.empid<150")
-        .explainContains("EnumerableCalc(expr#0..4=[{inputs}], empid=[$t0], "
-            + "name=[$t2], dept=[$t4])\n"
+        .explainContains(""
+            + "EnumerableCalc(expr#0..4=[{inputs}], empid=[$t0], name=[$t2], dept=[$t4])\n"
             + "  EnumerableHashJoin(condition=[=($1, $3)], joinType=[right])\n"
-            + "    EnumerableCalc(expr#0..4=[{inputs}], expr#5=[150], "
-            + "expr#6=[<($t0, $t5)], proj#0..2=[{exprs}], $condition=[$t6])\n"
-            + "      EnumerableTableScan(table=[[s, emps]])\n"
-            + "    EnumerableCalc(expr#0..3=[{inputs}], proj#0..1=[{exprs}])\n"
-            + "      EnumerableTableScan(table=[[s, depts]])\n")
+            + "    EnumerableCalc(expr#0..4=[{inputs}], expr#5=[CAST($t0):INTEGER NOT NULL], expr#6=[150], expr#7=[<($t5, $t6)], proj#0..2=[{exprs}], $condition=[$t7])\n"
+            + "      EnumerableTableScan(table=[[s, emps]])\n    EnumerableCalc(expr#0..3=[{inputs}], proj#0..1=[{exprs}])\n"
```

Comment:
if you are reformatting these plans you should start a new line after this newline.

Reply:
Reformatted.